validator/client/runner.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/prysmaticlabs/prysm/v5/config/params"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
 	"github.com/prysmaticlabs/prysm/v5/encoding/bytesutil"
-	"github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
+	prysmTrace "github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
```

Comment:
hmm this occurred again, why do we need both traces again?

Reply:
See line 229.